src/jsfeatNext.js

```diff
@@ -3,25 +3,33 @@ import data_type from './data_type/data_type.js'
 import cache from './cache/cache.js'
 import data_t from './node_utils/data_t.js'
 import keypoint_t from './keypoint_t/keypoint_t.js'
-import imgproc from './imgproc/imgproc.js'
-import math from './math/math.js'
+import { _resample, _resample_u8 } from './imgproc/resample.js'
+import { _convol, _convol_u8 } from './imgproc/convol.js'
+//import math from './math/math.js'
 import matmath from './matmath/matmath.js'
 import matrix_t from './matrix_t/matrix_t.js'
-import pyramid_t from './pyramid_t/pyramid_t.js'
-import linalg from './linalg/linalg.js'
-import orb from './orb/orb.js'
+//import pyramid_t from './pyramid_t/pyramid_t.js'
+//import linalg from './linalg/linalg.js'
+import { swap, hypot } from './linalg/linalg-base.js'
+//import orb from './orb/orb.js'
+import { bit_pattern_31 } from './orb/bit_pattern_31.js'
```

Comment:
this partially solve the orb example. I wa importing the bit_pattern_31 as with a default export (that's not correct) and solved the issue described in https://github.com/kalwalt/jsfeatNext/pull/4#issue-1304913132